owlbot.py

```diff
@@ -38,4 +38,17 @@
     templated_files, excludes=["docs/multiprocessing.rst", "noxfile.py", "CONTRIBUTING.rst"],
 )
 
+s.replace(
+    "docs/conf.py",
+    """\
+intersphinx_mapping = {
+    "python": ("https://python.readthedocs.org/en/latest/", None),
+""",
+    """\
+intersphinx_mapping = {
+    "python": ("https://python.readthedocs.org/en/latest/", None),
+    "requests": ("https://docs.python-requests.org/en/master/", None),
+""",
+)
+
```

Comment:
Hmm, owlbot seems not to be applying this change:  036499972586c97ca79fc9796fbc56b1bbd510a3 removes the entry for `"requests"`.  @parthea, @busunkim96 can you help?

Reply:
It looks like you can pass dict to `py_library` with additional intersphinx mappings:

https://github.com/googleapis/synthtool/blob/2430f8d90ed8a508e8422a3a7191e656d5a6bf53/synthtool/gcp/templates/python_library/docs/conf.py.j2#L370-L374

```py
common.py_library(
    ...,
    intersphinx_dependencies={
        "requests": "https://docs.python-requests.org/en/master/"
    }
)
```

Not sure why the current replacement is getting undone though.

Reply:
I needed to escape the parens in the text-to-be-replaced. :)